drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -2113,3 +2113,19 @@ int bnxt_hwrm_func_bw_cfg(struct bnxt *bp, uint16_t vf,
 
 	return rc;
 }
+
+int bnxt_hwrm_func_cfg_set_vf_mac_spoof(struct bnxt *bp, uint16_t vf,
```

Comment:
This should just be something like bnxt_hwrm_func_cfg_set_flags() and not take a flags argument.